server/pages/index.js

```diff
@@ -391,12 +394,16 @@ module.exports = function() {
 				: 'all';
 
 			if ( config.isEnabled( 'server-side-rendering' ) ) {
+				const store = createReduxStore();
+				store.dispatch( setSection( 'design', { hasSidebar: false } ) );
+				context.store = pick( store.getState(), 'ui' );
```

Comment:
Conversation from #2786:

> Why not bootstrap the whole store?

Because the themes stores use immutable maps, so we'd have to use fromJS() on the theme parts when creating the store on the client. I suggest we tackle that when we have some theme data to bootstrap.